index.html

```diff
@@ -6,8 +6,38 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Document</title>
     <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.2/css/all.min.css" integrity="sha512-1sCRPdkRXhBV2PBLUdRb4tMg1w2YPf37qatUFeS7zlBy7jJI8Lf4VHwWfZZfpXtYSLy85pkm9GaYVYMfw5BC1A==" crossorigin="anonymous" referrerpolicy="no-referrer" />
 </head>
 <body>
-    
+    <!-- Header Section -->
+    <Header>
+        <nav class="navbar">
+            <div class="logo">
+                <div class="logo-text">Zied</div>
+            </div>
+            <img src="images/align-right.svg" alt="Menu-icon">
+        </nav>
+    </Header>
+    <!-- Main Section -->
+    <main>
+        <div class="container">
+            <div class="blur-background">
+            <h1><span class="name-span">I'm Zied</span> <br/> Glad to see you! </h1>
+            <p id="header-description">I'm a software developer! I can help you build product, feature or website look through some of my work and experience! 
+                if you like what you see and have a project you need coded, don't hesitate to contact me.
+            </p>
+            <span><a class="link-connect" href="#">Let's Connect</a></span>
+            <div class="social-icon">
+                <i class="fa-brands fa-square-github"></i>
+                <i class="fa-brands fa-linkedin"></i>
+                <i class="fa-brands fa-square-twitter"></i>
+            </div>
```

Comment:
- [ ] Kindly use the `ul` tag to group your icons, and `li `per icon; each icon has a link `<a>` tag. This is a project requirement.